dissect/target/plugins/os/unix/ssh.py

```diff
@@ -115,45 +131,77 @@ def known_hosts(self):
                     )
 
     @export(record=PrivateKeyRecord)
-    def private_keys(self):
+    def private_keys(self) -> Iterator[PrivateKeyRecord]:
+        """Yields OpenSSH private keys on a Unix target per user."""
+        private_key_files = []
+
         for user_details in self.target.user_details.all_with_home():
             for file_path in user_details.home_path.glob(".ssh/*"):
-                file_buf = file_path.read_bytes()
+                private_key_files.append((user_details.user.name, file_path))
 
-                if b"PRIVATE KEY-----" not in file_buf:
-                    continue
+        for file_path in self.target.fs.path("/etc/ssh/").glob("*.pub"):
```

Comment:
Shouldn't this be `.glob("*")`?

Reply:
Correct, fixed in [03f9cd1](https://github.com/fox-it/dissect.target/pull/182/commits/03f9cd1bac255710b4e92c6260b08f9b71c94810).